src/main/java/core/selenium/webdrivers/FactoryBrowser.java

```diff
@@ -0,0 +1,26 @@
+package core.selenium.webdrivers;
+
+/**
+ * FactoryBrowser class.
```

Comment:
Plese be more descriptive in your comment.

Reply:
Thank you, this wrong was fixed.